content/doc/book/pipeline/getting-started.adoc

```diff
@@ -272,7 +272,7 @@ on the plugins installed which explicitly expose steps for use in Pipeline.
 
 To generate a step snippet with the Snippet Generator:
 
-. Navigate to the *Pipeline Syntax* link (referenced above) from a configured Pipeline, or at link:http://localhost:8080/pipeline-syntax[localhost:8080/pipeline-syntax].
+. Navigate to the *Pipeline Syntax* link (referenced above) from a configured Pipeline, or at link:http://localhost:8080/pipeline-syntax[localhost:8080/pipeline-syntax], assuming you have a Jenkins instance running on localhost port 8080.
```

Comment:
```suggestion
. Navigate to the *Pipeline Syntax* link (referenced above) from a configured Pipeline, or at `${YOUR_JENKINS_URL}/pipeline-syntax`.
```

Reply:
Agreed.  Thanks.  I've included it in c098a319493481a7a9c6433af912461393eccf89